active_tcp.go

```diff
@@ -18,7 +18,7 @@ import (
 type activeTCPConn struct {
 	readBuffer, writeBuffer *packetio.Buffer
 	localAddr, remoteAddr   atomic.Value
-	closed                  int32
+	closed                  atomic.Int32
```

Comment:
Would it be possible to use `atomic.Bool`?

Reply:
It is indeed possible to use `atomic.bool`